src/components/App.js

```diff
@@ -20,6 +20,11 @@ const App = ({ isOfficeInitialized }) => {
     setActiveNav(name);
   };
 
+  /**
+   * Change the startup state of the add in
```

Comment:
Add a full stop

Reply:
Be more descriptive.

`Set whether add-in should load by default on MS Word startup.`